src/components/GameView/GameOverlays.vue

```diff
@@ -191,16 +224,45 @@ export default {
 </script>
 
 <style scoped lang="scss">
+:deep(.v-overlay__content) {
+  left: 0;
+}
```

Comment:
I'm not sure why this is necessary but the overlay content was misaligning with the window without it